core/src/main/scala/kafka/zk/ZkData.scala

```diff
@@ -458,14 +459,14 @@ object StateChangeHandlers {
   */
 case class ZkAclStore(nameType: ResourceNameType) {
   val aclPath: String = nameType match {
-    case Literal => "/kafka-acl"
-    case Prefixed => "/kafka-prefixed-acl"
+    case ResourceNameType.LITERAL => "/kafka-acl"
+    case ResourceNameType.PREFIXED => "/kafka-prefixed-acl"
     case _ => throw new IllegalArgumentException("Unknown name type:" + nameType)
```

Comment:
Probably better to say "invalid name type" since rather than "Unknown name type", since the name type may not be UNKNOWN